cli/orchestration/scripts.md

```diff
@@ -86,6 +86,8 @@ script "command" "subcommand" { # any level of subcommands is supported
 
   - `name` _(optional)_ - A name for the jobs being executed (maximum of 128 chars)
   - `description` _(optional)_ - A description of the jobs being executed (maximum of 1000 chars)
+  - [`lets`](../code-generation/variables/lets.md) _(optional)_ - One or more blocks containing local variable definitions for the script.
```

Comment:
@soerenmartius see, this is weird, `globals` and `lets` are not a code generation only variables.